lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.js

```diff
@@ -206,6 +206,12 @@ testRule({
 			description: 'autofixer should not mangle css functions with comma separated values',
 			message: messages.expected('transition'),
 		},
+		{
+			code: 'a { border-top-width: 0px; border-right-width: 1px; border-bottom-width: 2px; border-left-width: 3px; }',
+			fixed: 'a { border-width: 0px 1px 2px 3px; }',
+			description: 'explicit border-width test',
+			message: messages.expected('border-width'),
+		},
 	],
```

Comment:
Can you add an example for `margin` or `border-width` or `padding` for `top | left/right | bottom`.
If if it's not supported yet add `skip: true,`.

Reply:
Sorry, a bit confused -- what do you mean by `top | left/right | bottom`? Do you mean when all four are specified, or when some of the values are inferred/collapsed? The latter isn't covered by this rule.

Reply:
> The latter isn't covered by this rule.

That's why I was proposing to add `skip: true` for now.
It could be an enhancement; it's expected by users.
cf #6947